### PR TITLE
Remove some notable strats in Maridia

### PR DESCRIPTION
**Pull request by kjbranch**

### CODE REVIEW

**Review by osse101 — COMMENTED** [review]
Puyo wrangling is a pain.  Sometimes they decide to not go where you need them.

Getting closer to just OKing deep stuck xray climbs.  Just not too sure where those work other than these places that aren't working with regular stuck.

File: region/maridia/inner-green.json [hunk 1]
```diff
@@ -1232,7 +1235,8 @@
                     "Grapple",
                     "canSuitlessMaridia",
                     "HiJump",
-                    "canFlatleyJump"
+                    "canFlatleyJump",
+                    "canConsecutiveWalljump"
```
[the remark this answers is not in the document]
Comment (kjbranch) [reply region/maridia/inner-green.json:1239]: The video I saw of this didn't use the trick where you don't alternate walls. It only took a few wall jumps. I couldn't decide if it should require that

File: region/maridia/inner-pink.json [hunk 2]
```diff
@@ -265,24 +265,25 @@
...
                 },
                 {
                   "name": "Crab Shaft Bottom Frozen Crab Steps",
-                  "notable": true,
+                  "notable": false,
                   "requires": [
                     "canSuitlessMaridia",
-                    "canUseFrozenEnemies"
+                    "canTrickyUseFrozenEnemies",
+                    "h_canCrouchJumpDownGrab"
```
[the remark this answers is not in the document]
Comment (kjbranch) [reply region/maridia/inner-pink.json:286]: Not sure what the frozen enemy boost is. I'm assuming it's the one on the ceiling to get onto it?

File: region/maridia/inner-yellow.json [hunk 3]
```diff
@@ -835,11 +857,19 @@
...
+                        "canSunkenTileWideWallClimb"
+                      ]}
+                    ]}
                   ],
                   "note": [
-                    "Doesn't actually require stepping on two successive Skulteras.",
-                    "Doable by standing on the left edge of the top ledge and crouch jump + downgrabbing on the top fish."
+                    "Use the Skultera to get onto the top right platform. There are two distinct ways to get up using the top Skultera.",
+                    "1. Freeze it at the bottom of its path to the left. Crouch jump and down grab to get onto it. The positioning is very precise.",
+                    "2. Freeze it about a half-tile from the right wall and very quickly stationary spin jump and walljump mash to get up before it thaws."
```
[the remark this answers is not in the document]
Comment (kjbranch) [reply region/maridia/inner-yellow.json:872]: I'm not seeing how to do this

File: region/maridia/inner-green.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (osse101) [reply region/maridia/inner-green.json]: Alternating sides is always the harder way, which actually makes deciding the tech difficulty interesting.

File: region/maridia/inner-pink.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (osse101) [reply region/maridia/inner-pink.json]: Rubbing a frozen enemy to gain height isn't a tech but more like an assumed part of underwater movement?  But its also not known by that many people really.  It's in a weird spot.  And its something your going to accidently do and not notice too.
And yes, its the top crab.  As soon as you can collide with the side of it, do so.

File: region/maridia/inner-yellow.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (osse101) [reply region/maridia/inner-yellow.json:882]: `canSunkenTileWideWallClimb` Has you face one direction when walljumping the whole time.  You want to jump against the wall into the fish to gain more height with each jump.
I was talking specifically about 2.

**Pull request merged**